Travel Scheduler/Helper Files/TravelSchedulerHelper.h

```diff
@@ -33,6 +33,27 @@ typedef NS_ENUM(NSInteger, DayOfWeek)
     DayOfWeekFriday,
     DayOfWeekSaturday
 };
+    
+typedef NS_ENUM(int, CustomeColor)
```

Comment:
I think it should be spelled CustomColor. Also, for enums, the general format should be adding the enum title to the names of the variables so LightGreenishBlue should be CustomColorLightGrenishBlue and so on.

Reply:
Oh got it! Thanks for the advice :)

Reply:
Done!